packages/block-library/src/latest-posts/edit.native.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * External dependencies
+ */
+import { TouchableWithoutFeedback, View, Text } from 'react-native';
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose, withPreferredColorScheme } from '@wordpress/compose';
+import { withDispatch } from '@wordpress/data';
+import { coreBlocks } from '@wordpress/block-library';
+import { __ } from '@wordpress/i18n';
+import { postList as icon } from '@wordpress/icons';
+import { InspectorControls } from '@wordpress/block-editor';
+import { fetchRequest } from 'react-native-gutenberg-bridge';
```

Comment:
This shouldn't be used directly and instead, it should use the GB `apiFecth` interface. This will provide support for retries and cache of the data.

The only thing needed is to add the `/wp/v2/categories` to the white list here: https://github.com/wordpress-mobile/gutenberg-mobile/blob/e288c45e882db9808e0396715f379edb07f32416/src/api-fetch-setup.js#L44

@chipsnyder if you need help please ping me or @etoledom.

Reply:
Thanks @SergioEstevao! I'll make a PR either today or tomorrow morning for that and push it up.